roles/app/tasks/docker.yml

```diff
@@ -61,6 +71,8 @@
       ACCESS_TOKEN_ENC_SECRET: "{{ access_token_secret }}"
       AMQP_URL: "{{ amqp_protocol }}://{{ rabbitmq_username }}:{{ rabbitmq_password }}@{{ rabbitmq_host }}/{{ rabbitmq_vhost | urlencode | replace('/', '%2F') }}"
       DATABASE_URL: "ecto://{{ wanda_postgres_user }}:{{ wanda_postgres_password }}@{{ wanda_postgres_host }}/{{ wanda_postgres_db }}"
+    volumes_from:
+      - "{{ checks_container_name }}"
```

Comment:
nitpick: maybe we can mount as read only:
```
- "{{ checks_container_name }}:ro"
```

Reply:
Here we go 6361b1d027a44321fab25997606cbc56894833d0